src/leiningen/github_cdn.clj

```diff
@@ -63,23 +66,23 @@
           (println "Removing old files")
           (git "rm" "-rf" "*" {:dir *dir* :throw false})
           (copy-files src-dir *dir*)
-          (println "Commiting and pushing")
+          (println "Committing and pushing")
           (git "add" "*" {:dir *dir* :throw false})
           (git "commit" "-m" comment' {:dir *dir* :throw false})
           (git "push" "origin" (str branch ":" branch) {:dir *dir*})
           (println "Done")))
       (println "No source dir specified"))))
 
 (defn github-cdn
-  "Publishes resources to gihhub branch"
+  "Publishes resources to gh-pages branch"
```

Comment:
You have it here `(println "Branch" branch "already exists")` which is very good for console output so you see what's going on. But I don't think it's a good idea for a commit message cause e.g. you have `gh-pages` it will comment `Updated files on gh-pages branch` and then you want to move the repo to `company.github.io` root and you push everything to `master` where history will still have `Updated files on gh-pages branch` which makes not a lot of sense in my opinion. Better to have static and neutral - `Updated files`, `Updated files with git-cdn`, `Updated files on github` etc